pkg/scannerv4/mappers/mappers.go

```diff
@@ -695,6 +762,47 @@ func pkgFixedBy(enrichments map[string][]json.RawMessage) (map[string]string, er
 	return pkgFixedBys, nil
 }
 
+// cveEPSS unmarshals and returns the EPSS enrichment, if it exists.
+func cveEPSS(ctx context.Context, enrichments map[string][]json.RawMessage) (map[string]epss.EPSSItem, error) {
+	enrichmentList := enrichments[epss.Type]
+	if len(enrichmentList) == 0 {
+		zlog.Warn(ctx).
+			Str("enrichments", epss.Type).
+			Msg("no EPSS enrichments found")
+		return nil, nil
+	}
+
+	var epssMap map[string][]epss.EPSSItem
+	err := json.Unmarshal(enrichmentList[0], &epssMap)
+	if err != nil {
+		return nil, fmt.Errorf("error unmarshaling EPSS enrichment: %w", err)
```

Comment:
no need to say `error` since this is an error. 
```suggestion
		return nil, fmt.Errorf("unmarshaling EPSS enrichment: %w", err)
```